app/models/event.rb

```diff
@@ -0,0 +1,14 @@
+class Event
+  rattr_initialize [
+    :address,
+    :city,
+    :ends_at,
+    :group_name,
+    :name,
+    :starts_at,
+    :state,
+    :url,
+    :group_name,
+    :venue_name,
```

Comment:
Is the order of these attributes significant?

If not, what do you think about sorting them alphabetically?

Reply:
just an oversight on my part, thanks! 🔤